app/src/main/java/com/codepath/chefster/adapters/FavoritesListAdapter.java

```diff
@@ -38,15 +40,29 @@ public FavoritesViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(FavoritesViewHolder holder, int position) {
-        Dish dish = dishesList.get(position);
+    public void onBindViewHolder(final FavoritesViewHolder holder, int position) {
+        final Dish dish = dishesList.get(position);
+
+        selectedDishesList = new ArrayList<>();
         holder.tvMealTitle.setText(dish.getTitle());
         holder.tvMealSummary.setText(dish.getDescription());
         holder.tvCookingTime.setText("Est." + String.valueOf(dish.getPrep_time() + dish.getCooking_time()) + " mins");
         holder.tvMealRating.setText(String.valueOf(dish.getRating()));
         if (dish.getThumbnails() != null && !dish.getThumbnails().isEmpty()) {
             Glide.with(context).load(dish.getThumbnails().get(0)).into(holder.ivMealImage);
         }
+
+
+        holder.cbSelectDish.setOnClickListener(new View.OnClickListener() {
+            @Override
```

Comment:
This works in the Adapter? Are you saving the list in the adapter? Why not save it in the fragment/activity and then add `ItemClickSupport` like in the guides that nathan gave us?